types/highcharts/index.d.ts

```diff
@@ -6487,9 +6487,10 @@ declare namespace Highcharts {
          * @param [boolean] redraw Whether to redraw the chart. Defaults to true.
          * @param [boolean] oneToOne When true, the series, xAxis and yAxis collections will be updated one to one, and
          * items will be either added or removed to match the new updated options. Defaults to false.
+         * @param [boolean] animation Whether to apply animation, and optionally animation configuration.
          * @since 5.0.0
          */
-        update(options: Options, redraw?: boolean, oneToOne?: boolean): void;
+        update(options: Options, redraw?: boolean, oneToOne?: boolean, animation?: boolean): void;
```

Comment:
It seems like this can also be an [`AnimationOptions](https://api.highcharts.com/class-reference/global#AnimationOptions) object. Can you add that as well?